workers-docs/src/content/reference/storage/_index.md

```diff
@@ -16,11 +16,10 @@ frequently read. Very infrequently read values are stored centrally, while
 more popular values are maintained in all of our data centers around the
 world.
 
-KV achieves this performance by being eventually-consistent. New key-value
-pairs are immediately available everywhere, but value changes may take up to
-60 seconds to propagate. Workers KV isn't ideal for situations where you need
-support for atomic operations or where values must be read and written in a
-single transaction.
+KV achieves this performance by being eventually-consistent. Changes may take
+up to 60 seconds to propagate. Workers KV isn't ideal for situations where
+you need support for atomic operations or where values must be read and
+written in a single transaction.
```

Comment:
We _may_ want to clarify that this is if you're writing from multiple places (such as the edge). You can get away with a lot more if you write from one service using the HTTP API, like many of our customers do for configuration data or whatever. Then again it's not guaranteed those would go to the same colo either...

Reply:
Yeah, my thinking was to encourage baseline expectations, even if you can sometimes do better.